editor/plugins/script_editor_plugin.cpp

```diff
@@ -4471,6 +4473,7 @@ ScriptEditor::ScriptEditor(WindowWrapper *p_wrapper) {
 
 		disk_changed_list = memnew(Tree);
 		disk_changed_list->set_hide_root(true);
+		disk_changed_list->set_accessibility_name(TTRC("The following files are newer on disk"));
```

Comment:
Does accessibility name auto-translate with auto-translation disabled?
Looking at the implementation, it doesn't. Probably needs TTR.

Reply:
No, it's only using `atr` so won't translate if it's disabled.